src/components/change-requests/change-requests.tsx

```diff
@@ -8,11 +8,18 @@ import { routes } from '../../shared/routes';
 import ChangeRequestsTable from './change-requests-table/change-requests-table';
 import ChangeRequestDetails from './change-request-details/change-request-details';
 import NewChangeRequest from './new-change-request/new-change-request';
+import ReviewChangeRequest from './review-change-request/review-change-request';
 import './change-requests.module.css';
 
 const ChangeRequests: React.FC = () => {
   return (
     <Switch>
+      <Route path={routes.CHANGE_REQUESTS_ACCEPT}>
+        <ReviewChangeRequest accepted={true} />
```

Comment:
```suggestion
        <ReviewChangeRequest accepted />
```
There's actually a shorthand where this ^ is equivalent to the code you wrote https://stackoverflow.com/questions/60816731/how-are-boolean-props-used-in-react